Update README.md (#2337)
rust-lang/rustup: README.md

* The **minimal** profile includes as few components as possible to get a
working compiler (`rustc`, `rust-std`, and `cargo`). It's recommended to use
this component on Windows systems if you don't use local documentation (the large

> number of files causes issues on that OS), and in

Reviewer: ```suggestion
number of files can cause issues with some AV systems), and in
```

Author: thanks for that @kinnison... can you spell it out

Reviewer: Can I spell what out?

Author: AV, to reduce the need to guess. Let me do it and squash the commits.
